test/e2e-secret-tests.sh

```diff
@@ -24,27 +24,32 @@ source $(dirname "${BASH_SOURCE[0]}")/e2e-common.sh
 readonly E2E_TEST_NAMESPACE="default"
 
 # Constants used for creating ServiceAccount for the Control Plane if it's not running on Prow.
-readonly CONTROL_PLANE_SERVICE_ACCOUNT_NON_PROW_KEY_TEMP="$(mktemp)"
+readonly CONTROL_PLANE_GSA_NON_PROW_KEY_TEMP="$(mktemp)"
 
-# Constants used for creating ServiceAccount for Data Plane(Pub/Sub Admin) if it's not running on Prow.
-readonly PUBSUB_SERVICE_ACCOUNT_NON_PROW_KEY_TEMP="$(mktemp)"
+# Constants used for creating ServiceAccount for the Sources if it's not running on Prow.
+readonly SOURCES_GSA_NON_PROW_KEY_TEMP="$(mktemp)"
+
+# Constants used for creating ServiceAccount for the Broker if it's not running on Prow.
+readonly BROKER_GSA_NON_PROW_KEY_TEMP="$(mktemp)"
```

Comment:
Do we need this at all? Can we just do sth like `readonly SOURCES_GSA_KEY_TEMP="$(mktemp)"`?

Reply:
No you are right. I had this as part of my initial PR but I thought we can do this in a separate PR. Anyways, I will include this.